src/main/java/gregtech/api/unification/material/Material.java

```diff
@@ -376,7 +377,7 @@ public Builder(int id, String name) {
 
         /**
          * Add a {@link FluidProperty} to this Material.<br>
-         * Will be created as a {@link FluidType#FLUID}, without a Fluid Block.
+         * Will be created as a {@link FluidTypes#LIQUID}, without a Fluid Block.
```

Comment:
This does not seem to work for all materials. Vanilla materials like Iron, Gold, Redstone, Glowstone,  is defined as a fluid with this method, but its tooltip does not reflect that it is a FluidTypes#Liquid.

In addition, Iron Plasma does not have the Plasma state tooltip, which may be related to this.

Reply:
Before this PR, these Materials had the state tooltips applied correctly.